qiskit/dagcircuit/__init__.py

```diff
@@ -25,6 +25,9 @@
 
    DAGCircuit
    DAGNode
+   OpNode
+   InNode
+   OutNode
```

Comment:
If possible, I think these would be better as `DAGOpNode`, `DAGInNode`, `DAGOutNode` so that they keep a bit more context in the class name. This would make e.g. error messages like `AttributeError: 'InNode' object has no attribute 'name'` a bit easier to parse and track down.

Reply:
Can do as a command line global replace.

Reply:
Done in https://github.com/Qiskit/qiskit-terra/pull/6567/commits/205ac2606600c88cf47ebc652623e1ca1fa68035